includes/admin/class-wc-admin-menus.php

```diff
@@ -151,6 +152,7 @@ public function addons_menu() {
 		/* translators: %s: extensions count */
 		$menu_title = sprintf( __( 'Extensions %s', 'woocommerce' ), $count_html );
```

Comment:
Let's use this variable for My Subscriptions menu item, as it contains the subscription update count.

Reply:
Thanks, updated it now 🙌